pyaerocom/units_helpers.py

```diff
@@ -129,10 +127,22 @@ def convert_unit(data, from_unit, to_unit, var_name=None):
     data
         data in new unit
     """
+    if isinstance(from_unit, str):
+        from_unit = Unit(from_unit)
+    if isinstance(to_unit, str):
+        to_unit = Unit(to_unit)
+    if from_unit == to_unit:
+        # nothing to do
+        return data
```

Comment:
Good idea! Do you know of this is using Unit().definition for checking equality?

Reply:
Not sure how exactly equality is implemented there but something like Unit('1/m') == Unit('m-1') will be True